src/pages/SubmissionDetails.js

```diff
@@ -124,9 +124,10 @@ export const SubmissionDetails = ({ submissionId }) => {
           onClick: () => Nav.goToPath('root')
         }, ['Submit a new workflow'])
       ]),
-      div({ style: { marginLeft: '4em' } }, [
+      div({ style: { marginLeft: '4em', lineHeight: 1.25 } }, [
```

Comment:
Seems a little suspicious that you'd need to manually edit line heights here? TOL maybe one day we can extract a style out of this so that all of the pages in the UI can benefit from the same style of lists?